microservices/the_monkeys_gateway/internal/user_service/routes.go

```diff
@@ -166,3 +167,14 @@ func (asc *UserServiceClient) DeleteUserProfile(ctx *gin.Context){
 	ctx.JSON(http.StatusOK, res)
 
 }
+
+func (asc *UserServiceClient) GetAllTopics(ctx *gin.Context){
+	res,err := asc.Client.GetAllTopics(context.Background(), &pb.GetTopicsRequests{})
+
+	if err!=nil {
+		ctx.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to delete user profile"})
```

Comment:
we have change the message as for now all topics are required

Reply:
@kalidasBattena  I have fixed it